automated/linux/fs-resize/fs-resize.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -e
```

Comment:
#!/bin/bash -e
Do we need this "-e" option ? what is the purpose this ?

Reply:
I guess you're right. Should not exit on error.

Reply:
fixed